virtcontainers/container.go

```diff
@@ -445,29 +445,45 @@ func (c *Container) shareFiles(m Mount, idx int, hostSharedDir, guestSharedDir s
 	caps := c.sandbox.hypervisor.capabilities()
 	if !caps.isFsSharingSupported() {
 		c.Logger().Debug("filesystem sharing is not supported, files will be copied")
+
+		fileInfo, err := os.Stat(m.Source)
+		if err != nil {
+			return "", false, err
+		}
+
+		// Ignore the mount if this is not a regular file (excludes
+		// directory, socket, device, ...) as it cannot be handled by
+		// a simple copy. But this should not be treated as an error,
+		// only as a limitation.
+		if !fileInfo.Mode().IsRegular() {
+			c.Logger().Debug("Ignore (%s), fs sharing not supported and not a regular file", m.Source)
```

Comment:
Nit: this and other log calls could be converted into structured format:

```go
c.Logger().WithField("ignored-file", m.Source).Debug("Ignoring non-regular file as FS sharing not supported")
```

Reply:
And it is causing CI failure:
```
# github.com/kata-containers/runtime/virtcontainers
virtcontainers/container.go:459: Entry.Debug call has possible formatting directive %s
FAIL	github.com/kata-containers/runtime/virtcontainers [build failed]
Makefile:421: recipe for target 'go-test' failed
```

Reply:
I fixed it for him and repushed.